index.html

```diff
@@ -265,6 +249,28 @@ <h2 class="contact-intro">
         <p>tylher123@gmail.com</p>
       </footer>
     </div>
-    <script src="script.js"></script>
+
+    <section class="modal-wrapper">
+      <div class="modal">
+        <div class="modal-image-box-desktop">
+          <a href="#" class="cancel-icon-desktop"><img src="images/Icon-Cancel-desktop.svg" alt=""></a>
+          <img class="project-image-desktop" src="images/Disabled.png" alt="" />
+        </div>
+        <div class="modal-text-box">
+          <h2></h2>
+          <ul class="modal-tech">
+          </ul>
+          <p>
+          </p>
+
+          <div class="btn-group">
+            <a class="seeLive" href="">see live <img src="images/Icon-GitHub.svg" alt="" /></a>
+            <a class="seeSource" href="">see source <img src="images/Icon-Export.png" alt="" /></a>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
Kindly change the color of the close button in the mobile version to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/156433453-c9277d10-ee73-43f1-b7ab-5aba4a913003.png) | ![image](https://user-images.githubusercontent.com/24703171/156433619-4ebdfdc4-f2ac-4a39-96a7-5ede16646561.png) |

- Kindly update details data based on project data. You filter the project based on names but all projects have the same name so kindly fix this.
![image](https://user-images.githubusercontent.com/24703171/156434289-6f89590d-ede0-4393-8189-200139afb748.png)

Reply:
Resolved the second part of the changes but for the first part the exit-icon is not visible because of the white background in the image